### DISCUSSION TITLE
Home page spacing

### DISCUSSION THREAD

**Discussion by PleaseStopAsking** [LeoKlaus/plappa#12] [Feature Requests]
With the new grid layout for the home tab (much appreciated by the way), the spacing is a little tight between each section. It could benefit from some spacing both below and above the title for each section. I can only test on an iPhone 14 Pro so it may be different on other devices however.

**Comment by LeoKlaus** [comment]
The padding is fairly similar among all devices. I agree that it should be a bit bigger, I'll definitely change this with the next release (which may take a while, as I'm working on some bigger internal changes).

**Comment by LeoKlaus** [comment]
I've added some padding with the latest beta release, closing this now.
